tutorials/Tutorial MIS & QAOA.ipynb

```diff
@@ -0,0 +1,394 @@
+    "finance, or broadcast systems (wifi or cellular network) optimization."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For example, assume an ensemble of identical radio transmitters over French cities that each have the same radius of transmission. It was quickly realized that two transmitters with close or equal frequencies could interfere with one another, hence the necessity to assign non-interfering frequencies to overlapping transmiting towers. Because of the limited amount of bandwith space, some towers have to be assigned the same or close frequencies. The MIS of a graph of towers indicate the maximum number of towers that can have close or equal given frequency (red points). \n",
+    "\n",
+    "<img src=\"files/France_MIS.png\" alt=\"MIS map France\" style=\"width: 700px;\"/>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we will show how to solve the MIS problem on Unit-Disk (UD) graphs with an excellent approximation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When looking for the MIS of a graph, we separate the nodes into two distinct classes: an independence one and the others. We can attribute a status $z$ to each node, where $z_i = 1$  if node $i$ is attributed to the independent set, and $z_i=0$ otherwise. The Maximum Independent Set corresponds to  the minima of the following cost function: \n",
+    "\n",
+    "\\begin{equation}\n",
+    "   C(z_1,\\dots,z_N) = -\\sum_{i=1}^N z_i + U \\sum_{\\langle i,j \\rangle}z_i z_j\n",
+    " \\label{cost_function}\n",
+    "\\end{equation}\n",
+    "where $U \\gg 1$  and $\\langle i,j \\rangle$ represents adjacent nodes (i.e. there is a link between node $i$ and $j$). In this cost function, we want to promote a maximal number of atoms to the $1$ state, but the fact that $U \\gg 1$  strongly penalizes two adjacent vertices in state $1$. The minimum of $C(z_0,\\dots,z_N)$ therefore corresponds to the maximum independent set of the graph."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Interestingly, the operator $\\hat{C}$ associated with the cost function of the previous equation can be natively realized on a neutral atom platform with some constraints on the graph edges. We map a  ground state and a Rydberg state of each atom to a spin $1/2$, where $|0 \\rangle = |r \\rangle$ is a Rydberg state and $|1 \\rangle = |g \\rangle$ is a ground state. An atom in a Rydberg state has an excited electron with a very high principal quantum number and therefore exhibits a huge electric dipole moment. As such, when two atoms are excited to Rydberg states, they exhibit a strong van der Waals interaction. Placing $N$ atoms at positions $\\textbf{r}_j$ in a 2D plane, and coupling the ground state $|1\\rangle$ to the Rydberg state $|0\\rangle$ with a laser system enables the realization of the Hamiltonian :\n",
```

Comment:
I'd change "spin 1/2" to "two-level system" to simplify the picture. Also, since we will be sampling in the `ground-rydberg` basis, you should put |0> = |g> and |1> = |r>. Then, remark that a `0` in the bistring corresponds to a `1` in our sampling.

Another option is to go on with the |1> = |g> and |0> = |r> labelling, and then "flip" the results of the sampling  so that they agree with your convention.

Reply:
replaced as "two-level system" with |0> = |g> and |1> = |r>. In that case, I do not believe that a `0` in a bit-string corresponds to a `1` in our sampling: the initial state is `00000` and corresponds to the ground-state.